content/docs/setup/kubernetes/helm-install/index.md

```diff
@@ -86,282 +86,38 @@ to manage the lifecycle of Istio.
-| `tracing.service.internalPort` | `9411` |  | `` |
-| `tracing.service.uiPort` | `16686` |  | `` |
-| `tracing.ingress` | `zipkin.local` |  | `` |
-| `tracing.ingress.enabled` | `false` |  | `` |
-| `tracing.ingress.hosts` | `zipkin.local` |  | `` |
-| `tracing.ingress.annotations` | `` |  | `` |
-| `tracing.ingress.tls` | `` |  | `` |
-| `tracing.resources` | `{}` |  | `` |
-| `kiali` | `` |  | `` |
-| `kiali.enabled` | `false` |  | `` |
-| `kiali.replicaCount` | `1` |  | `` |
-| `kiali.image` | `` |  | `` |
-| `kiali.image.repository` | `kiali/kiali` |  | `` |
-| `kiali.image.tag` | `0.3.1.Alpha` |  | `` |
-| `kiali.ingress` | `` |  | `` |
-| `kiali.ingress.enabled` | `false` |  | `` |
-| `kiali.ingress.annotations` | `` |  | `` |
-| `kiali.ingress.tls` | `` |  | `` |
-| `kiali.resources` | `{}` |  | `` |
-| `kiali.dashboard` | `` |  | `` |
-| `kiali.dashboard.username` | `admin` |  | `` |
-| `kiali.dashboard.password` | `admin` |  | `` |
-<!-- AUTO-GENERATED-END -->
-
-The Helm chart also offers significant customization options per individual
-service. Customize these per-service options at your own risk. The per-service options are exposed via
-the [`values.yaml`]({{< github_file >}}/install/kubernetes/helm/istio/values.yaml) file.
-
-## Customization example: traffic management minimal set
-
-Istio is equipped with a rich and powerful set of features and some users may need only subset of those. For instance, users might be interested only
-in installing the minimal set required to Istio's traffic management.
-[Helm customization](#customization-with-helm) provides the option to install a subset by enabling those features of interest and disabling the ones that aren't required.
-
-In this example we will install Istio with only a minimal set of components necessary to conduct [traffic management](/docs/tasks/traffic-management/).
-
-Execute the following command to install the Pilot, Citadel, IngressGateway and Sidecar-Injector:
+## Customization Example: Traffic Management Minimal Set
+
+Istio has a rich feature set, but you may only want to use a subset of these. For instance, you might be only interested in installing the minimum necessary to support traffic management functionality.
+
+This example shows how to install the minimal set of components necessary to use [traffic management](/docs/tasks/traffic-management/) features.
+
+Execute the following command to install the Pilot and Citadel:
 
 {{< text bash >}}
 $ helm install install/kubernetes/helm/istio --name istio --namespace istio-system \
-  --set ingress.enabled=false,gateways.istio-egressgateway.enabled=false,galley.enabled=false \
-  --set mixer.enabled=false,prometheus.enabled=false,global.proxy.envoyStatsd.enabled=false
+  --set ingress.enabled=false \
+  --set gateways.istio-ingressgateway.enabled=false \
+  --set gateways.istio-egressgateway.enabled=false \
+  --set galley.enabled=false \
+  --set sidecarInjectorWebhook.enabled=false \
```

Comment:
Rather than al la carte as the name, I'd use

"Installing Istio without Mixer" and show how that is done (meaning the webhook should be enabled).

Or another option is to just add that mode in.

I think in general we should provide 5-7 examples to create istio helm charts  that do meaningful things for people.

My main attraction to this model is to enable things like "How do I install Istio without Mixer?".  I'm sure folks will come up with other ideas as time passes.

Cheers
-steve